region/lowernorfair/east.json

```diff
@@ -3148,7 +3416,23 @@
                     "h_canNavigateHeatRooms",
                     "h_canUseMorphBombs",
                     {"heatFrames": 500}
-                  ]
+                  ],
+                  "obstacles": [
+                    {
+                      "id": "B",
+                      "requires": [
+                        {"or": [
+                          "ScrewAttack",
+                          {"and": [
+                            "canCarefulJump",
+                            {"heatFrames": 600}
+                          ]},
+                          { "enemyDamage": { "enemy": "Kihunter (red)", "type": "contact", "hits": 1 } }
+                        ]}
+                      ]
+                    }
+                  ],
+                  "note": "Stand on the watchpost for the KiHunter to move right before jumping up to get a safe predicatable swoop."
```

Comment:
Not sure what the watchpost is referring to. The screw attack/tank variants wont require waiting - The tank variant could be the best option, for instance with gravity alone, and map rando's suits.

Reply:
600 isn't right... more like 150